### PR TITLE
Added shortcuts

### PR DESCRIPTION
**Pull request by Variable-ind**
Added *almost* each and every existing Shortcut in the Preferences Dialog (Also i removed one of the two fullscreen shortcuts in favor of this)

### CODE REVIEW

File: src/Autoload/Tools.gd [hunk 1]
```diff
@@ -95,7 +95,7 @@ var tools := {
 		"""Hold %s to snap the angle of the line
 Hold %s to center the shape on the click origin
 Hold %s to displace the shape's origin""",
-		["Shift", "Ctrl", "Alt"]
+		["Shift", "configurable_ctrl", "configurable_alt"]
```
Comment (OverloadedOrama) [inline src/Autoload/Tools.gd:98]: Shouldn't this have "configurable_shift" instead of "Shift"?

Comment (Variable-ind) [reply src/Autoload/Tools.gd:98]: No because that shift's not influenced by input map and isn't configurable

Comment (Variable-ind) [reply src/Autoload/Tools.gd:98]: the "shift" represented here uses "event.shift" from "InputEventWithModifiers" (instead of the "shift" from "input map")

**Review by OverloadedOrama — APPROVED** [review]
Thank you!

**Pull request merged**